findoutlie/metrics.py

```diff
@@ -29,4 +31,17 @@ def dvars(img):
     # You may be be able to solve this in four lines, without a loop.
     # But solve it any way you can.
     # This is a placeholder, replace it to write your solution.
+
+    return np.sqrt(np.mean((np.diff(img.get_fdata()))**2, axis=(0,1,2)))
```

Comment:
I would say this is too compact, making it harder to see what the code is doing at quick glance - and a quick glance is often all you get.  For example, it would be very easy to mess up the parentheses here.   And you have to know that `axis=-1` is the default for `np.diff` (I didn't).

```suggestion
    data = img.get_fdata())
    ddif = np.diff(data, axis=-1)   # Make default explicit.
    return np.sqrt(np.mean(ddiff ** 2, axis=(0,1,2)))
```